tests/BaseTest.php

```diff
@@ -0,0 +1,62 @@
+<?php namespace EightshiftFormsTests;
+
+use \Brain\Monkey;
+use \Brain\Monkey\Functions;
+
+class BaseTest extends \Codeception\Test\Unit
+{
+
+  const WP_REDIRECT_ACTION = 'eightshift_forms_test/wp_safe_redirect_happened';
+  const WP_MAIL_ACTION = 'eightshift_forms_test/wp_mail_happened';
```

Comment:
is this eightshift_forms_test correct?

Reply:
Yup, these are actions **specifically for unit tests** which we can use to see if some native WP function happened that would be otherwise hard to test (such as `wp_mail()` and `wp_safe_redirect()` in this case) .